app/views/reviews/edit.html.erb

```diff
@@ -8,3 +8,7 @@
 
   <%= f.submit 'Edit Review' %>
 <% end %>
+
+
+<%= button_to "Delete Review", {:controller => :reviews,
+      :action => 'destroy', :id => @review.id }, :method => :delete %>
```

Comment:
Im not seeing the associated testing for this feature. Please update if they dont exist, or direct in the right direction

Reply:
@nwalberts - We need to write those tests! Will update you when done.